src/main/drivers/rx/rx_pwm.c

```diff
@@ -139,15 +142,15 @@ typedef struct ppmISREvent_s {
 static ppmISREvent_t ppmEvents[20];
 static uint8_t ppmEventIndex = 0;
 
-void ppmISREvent(eventSource_e source, uint32_t capture)
+static void ppmISREvent(eventSource_e source, uint32_t capture)
 {
     ppmEventIndex = (ppmEventIndex + 1) % ARRAYLEN(ppmEvents);
 
     ppmEvents[ppmEventIndex].source = source;
     ppmEvents[ppmEventIndex].capture = capture;
 }
 #else
-void ppmISREvent(eventSource_e source, uint32_t capture) {}
+static void ppmISREvent(eventSource_e source, uint32_t capture) {}
```

Comment:
Since the `ppmISREvent`function is now defined on line 133,  do we need to re-define it on lines 145 and 153?

Reply:
Line 133 is prototype (to enforce that functions in `#if` and `#else` branches are identical). 145 is DEBUG_PPM_ISR, 153 is `#else` dummy function.